### PR TITLE
Speed up some travis builds

### PR DESCRIPTION
**Pull request by sgiehl**
skips not required setup parts

refs #6312

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -50,13 +50,20 @@ matrix:
     - php: 5.6
       env: TEST_SUITE=UnitTests MYSQL_ADAPTER=PDO_MYSQL
       sudo: false
+      addons: false
```
Comment (sgiehl) [inline .travis.yml]: skips installing any apt packages

File: .travis.yml [hunk 2]
```diff
@@ -50,13 +50,20 @@ matrix:
...
       sudo: false
+      addons:
+        apt:
+          packages:
+            - nginx
+            - realpath
     - php: 5.6
       env: TEST_SUITE=AngularJSTests MYSQL_ADAPTER=PDO_MYSQL SKIP_COMPOSER_INSTALL=1
       sudo: false
+      addons: false
```
Comment (sgiehl) [inline .travis.yml]: skips installing any apt packages

File: .travis.yml [hunk 3]
```diff
@@ -50,13 +50,20 @@ matrix:
...
     # Javascript tests
     - php: 5.6
       env: TEST_SUITE=JavascriptTests MYSQL_ADAPTER=PDO_MYSQL
       sudo: false
+      addons:
+        apt:
+          packages:
+            - nginx
+            - realpath
```
Comment (sgiehl) [inline .travis.yml]: only install apt packages that are required

File: .travis.yml [hunk 4]
```diff
@@ -105,27 +112,6 @@ before_script:
...
-
-  - mysql -e "SELECT @@sql_mode;"
-  # - mysql -e "SHOW GLOBAL VARIABLES;"
-
-  # print out more debugging info
-  - uname -a
-  - date
-  - php -r "var_dump(gd_info());"
-  - mysql -e 'create database piwik_tests;'
-
```
Comment (sgiehl) [inline .travis.yml]: This block was moved to travis scripts

**Comment by diosmosis** [comment]
This & matomo-org/travis-scripts#39 both look good to me

**Pull request merged**